addons/web/static/src/js/views/kanban/kanban_record.js

```diff
@@ -87,6 +87,22 @@ var KanbanRecord = Widget.extend(WidgetAdapterMixin, {
     // Public
     //--------------------------------------------------------------------------
 
+    /**
+     * Check if a html content is empty. If there are only formatting tags with
+     * style attributes or a void content. Famous use case is
+     * '<p style="..." class=".."><br></p>' added by some web editor(s).
+     * Note that because the use of this method is limited, we ignore the cases
+     * like there's one <img> tag in the content. In such case, even if it's the
+     * actual content, we consider it empty.
+     *
+     * @param {string} htmlContent
+     * @returns {boolean} true if no content found or if containing only formatting tags
+     */
+    isHtmlEmpty: function (htmlContent) {
+        let div = document.createElement('div');
+        div.innerHTML = htmlContent || "";
+        return div.innerText.trim() === "";
+    },
```

Comment:
Looks quite suspicious. I'm sure there's a cleaner way to achieve this :)

Reply:
Hello @tivisse ,

It was suggested by GED on the task chatter